pyproject.toml

```diff
@@ -24,22 +26,25 @@ extend-safe-fixes = ["UP007", "UP006"]
 
 ignore = [
     "C90",  # mccabe
-    "FBT", # allow boolean args.
+    "FBT",  # allow boolean args.
     "CPY",  # copyright
     "S101", # allow asserts.
-    "E731",  # Allow lambda
-    "E741",  # Allow short variable names.
-    "D102",  # Do not check public method docstrings (Cannot determine overridden docstrings)
-             # @typing.override not available until python 3.12 (PEP 698)
-    "PLR",  # Too many of too many checks.
-    "ANN101",  # Ignore self type annotation.
+    "E731", # Allow lambda
+    "E741", # Allow short variable names.
+    "D102", # Do not check public method docstrings (Cannot determine overridden docstrings)
```

Comment:
I think should not be globally overridden as we may miss docstrings in the future.

Reply:
this is not added by me though ……